src/index.html

```diff
@@ -22,6 +22,361 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__logo logo">
+        <a
+          href="#"
+          class="logo__link"
+        >
+          <img
+            src="./images/logo.png"
+            alt="Logo"
+          />
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__list-item">
+            <a
+              href="#"
+              class="is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a href="#">Samsung</a>
+          </li>
+          <li class="nav__list-item">
+            <a href="#">Smartphones</a>
+          </li>
+          <li class="nav__list-item">
+            <a
+              href="#"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a href="#">Gadgets</a>
+          </li>
+          <li class="nav__list-item">
+            <a href="#">Tablets</a>
+          </li>
+          <li class="nav__list-item">
+            <a href="#">Photo</a>
+          </li>
+          <li class="nav__list-item">
+            <a href="#">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="catalog">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          src="./images/imac.jpeg"
+          alt="iMac 27 Retina"
+          class="card__image"
+        />
+
+        <div class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+
+        <div class="card__code">Product code: 195434</div>
+
+        <div class="card__stars">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="reviews">Reviews: 5</div>
+        </div>
+
+        <div class="card__price">
+          <div class="card__price-text">Price:</div>
+          <div class="card__price-num">$2,199</div>
+        </div>
+
+        <a
+          href="https://www.moyo.ua/ua/"
+          class="card__button"
+          data-qa="card-hover"
```

Comment:
The `data-qa` attribute for the 'Buy' link inside the first card should be corrected from `data-qa="hover"` to `data-qa="card-hover"` to align with the task requirements.